systemtests/bgp_test.go

```diff
@@ -928,3 +928,20 @@ func (s *systemtestSuite) CheckBgpNoConnectionForaNode(c *C, node vagrantssh.Tes
 		}
 	}
 }
+
+func (s *systemtestSuite) CheckBgpRouteDistribution(c *C, node vagrantssh.TestbedNode, containers []*container) error {
+	count := 0
+	for {
```

Comment:
Can we break the infinite loop here with a really high timeout?
Else, this can loop forever when there is a route distribution bug